platformio/compat.py

```
@@ -146,3 +146,84 @@ def is_proxy_set(socks=False):
+    elif IS_WINDOWS:
+        if locate:
+            url = _unquote_file(url)
+            args = ["explorer", f"/select,{url}"]
+        else:
+            args = ["start"]
+            if wait:
+                args.append("/WAIT")
+            args.append("")
+            args.append(url)
+        try:
+            return subprocess.call(args, shell=True)
```

Ruff (0.14.8)

`subprocess` call with `shell=True` identified, security issue

(S602)